docs/authentication.md

```diff
@@ -112,6 +112,26 @@ When using `-AccessToken`, we won't have access to the refresh token and the cli
 
 Before using the provided `-AccessToken` to get Microsoft Graph resources, customers should ensure that the access token has the necessary scopes/ permissions needed to access/modify a resource.
 
+### Access Token Proof of Possession (AT PoP)
+
+AT PoP is a security mechanism that binds an access token to a cryptographic key that only the intended recipient has. This prevents unauthorized use of the token by malicious actors. AT PoP enhances data protection, reduces token replay attacks, and enables fine-grained authorization policies.
+
+Note: AT PoP requires WAM to function.
+
+Microsoft Graph PowerShell module supports AT PoP in the following scenario:
+
+- To enable AT PoP on supported devices
+
+```PowerShell
+Set-MgGraphOption -EnableATPoP $true
```

Comment:
Why would we create a whole new cmdlet for this instead of making it a parameter of Connect-MgGraph like all other auth options?

Reply:
- We want each user to be able to customize thier msgraph ps without having to run "Connect-MgGraph -EnableATPoP" or "Connect-MgGraph -WAM" everytime the user decides to either use WAM or the AT PoP feature
- We decided to store the user's settings per se in a json file named "MgGraphOption" which to my knowledge would include other setting option moving forward. This was initially done when implementing the WAM feature.

Reply:
Both WAM and POP should be the default options in my opinion. And the opt-out of these features obscure.

Reply:
I'm fine with this once we've proven the feature -- I'd like to avoid changing defaults until we've validated that we're not regressing or presenting user experience issues.

Reply:
Please consider #2790 before changing defaults.